lib/cmd/fh3/admin/policies/create.js

```diff
@@ -0,0 +1,88 @@
+/* globals i18n */
+var common = require("../../../../common");
+var fhreq = require("../../../../utils/request");
+var _ = require('underscore');
+var util = require('util');
+
+var validPolicies = [
+  'OAUTH1',
+  'OAUTH2',
+  'LDAP',
+  'OPENID',
+  'FEEDHENRY',
+  'MBAAS'
+];
+
+module.exports = {
+  'desc' : i18n._('Update a Auth Policy'),
+  'examples' :
+    [{
+      cmd : 'fhc admin policies create --id=<id> --type=<type> --checkUserExists=<true|false> --checkUserApproved=<true|false> --configurations=<configurations>',
+      desc : i18n._('Create the Auth Policy with the <id>,<type>,<checkUserExists>,<checkUserApproved> and <config>')
+    }],
+  'demand' : ['id'],
+  'alias' : {
+    'id' : 'i',
+    'type' : 't',
+    'checkUserExists' : 'ce',
+    'checkUserApproved' : 'ca',
+    'configurations' : 'c',
+    'json' : 'j',
+    0 : 'id',
+    1 : 'type',
+    2 : 'checkUserExists',
+    3 : 'checkUserApproved',
+    4 : 'configurations'
+  },
+  'describe' : {
+    'id' : i18n._('The Auth Policy ID'),
+    'type' : i18n._('The type of the Auth Policy'),
+    'checkUserExists' : i18n._('True for the Auth Policy check if exist the user and false to not check it'),
+    'checkUserApproved' : i18n._('True for the Auth Policy check if the user is approved and false to not check it'),
+    'configurations' : i18n._('It is a json object corresponding to the policy type. E.g ')
+    + i18n._('\n          for OAuth2: "{"clientId": "1234567890.apps.example.com",  "clientSecret": "Wfv8DQw80hhyaBqnW37x5R23", "provider": "GOOGLE"}"')
+    + i18n._('\n          for LDAP: "{"authmethod": "simple", "url": "ldap://foo.example.com:389, "dn": "ou=people,dc=example,dc=com", "dn_prefix": "cn", "provider": "LDAP"}')
+    + i18n._('\n                       authmethod can be one of: "simple", "DIGEST-MD5", "CRAM-MD5", or "GSSAPI"')
+    + i18n._('\n          for MBAAS: "{"provider":"MBAAS","mbaas":"mqijsw3tgpn6htwmbbg27jl6","authEndpoint":"/test","defaultEnvironment":"dev"}"'),
+    'json' : i18n._("Output in json format")
+  },
+  'customCmd': function(params, cb) {
+    if (params.type && !validPolicyType(params.type)) {
+      return cb(util.format(i18n._("'<type>' must be one of %s"), validPolicies.join(', ')));
+    }
+    if (params.configurations && typeof params.configurations === 'string') {
+      try {
+        params.configurations = JSON.parse(params.configurations);
+      } catch (x) {
+        return cb(i18n._("Error parsing 'configurations' parameter: ") + util.inspect(x));
+      }
+    }
+    var data = {
+      policyId:params.id,
+      policyType:params.type,
+      configurations:params.configurations || {},
+      checkUserExists:params.checkUserExists || false,
+      checkUserApproved:params.checkUserApproved || false
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Remove the literal "false" boolean value. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://metrics.skunkhenry.com/coding_rules#rule_key=javascript%3AS1125)